lib/dvl/components/flashes.rb

```diff
@@ -4,7 +4,7 @@ class Dvl::Components::Flashes < Erector::Widget
   needs :flash
 
   def content
-    @flash.select { |k, v| k.to_s.in?(%w(success error info)) && v.present? }.each do |k, v|
+    @flash.select { |k, v| k.to_s.in?(%w(success error info alert notice)) && v.present? }.each do |k, v|
```

Comment:
This is too many flash types. Where are we using `alert` and `notice`? We should be using `error` and `info` instead, respectively. 

I'm guessing that maybe these flashes are being handled by Devise? It looks like we can just override this method in our controllers, in that case: https://github.com/plataformatec/devise/blob/0a0b4e00ea9d4392afbdd01628acd86243001cff/app/controllers/devise_controller.rb#L147-L154

Let me know if I'm missing anything.